internal/authorizer/authorizer_test.go

```diff
@@ -45,9 +45,9 @@ func TestIsAllowedFromContext(t *testing.T) {
 		"Extract current user action from request": {action: myUserOtherAction, userUIDReturn: "1000", pid: 10000, uid: 1000, wantAuthorized: true},
 		"Extract other user action from request":   {action: myUserOtherAction, userUIDReturn: "999", pid: 10000, uid: 1000, wantAuthorized: true},
 
-		// Error cases
-		"Error when user lookup returns an error": {action: myUserOtherAction, userLookupError: true, pid: 10000, uid: 1000, wantAuthorized: false},
-		"Error when user has invalid uid":         {action: myUserOtherAction, userUIDReturn: "NaN", pid: 10000, uid: 1000, wantAuthorized: false},
+		// Unauthorized cases
+		"User lookup returns an error": {action: myUserOtherAction, userLookupError: true, pid: 10000, uid: 1000, wantAuthorized: false},
```

Comment:
I would add to the end something like "is unauthorized" to both tests while we are enhancing them.